docs/backends/spdk/index.rst

```diff
@@ -260,6 +231,25 @@ After doing this, then inspecting the configuration should output::
   /sys/devices/system/node/node0/hugepages/hugepages-2048kB/surplus_hugepages:0
 
 
+No devices found
+~~~~~~~~~~~~~~~~
```

Comment:
This is also duplicated from the backends section

Reply:
Please skip the backends and tools sections, as they are incomplete and may not make sense. These sections were not addressed due to the substantial size of this PR.